shared/psalm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
And a psalm.xml :) Looks like you probably just need to add shared/ to the run-composers target in the Makefile so the vendor folder is populated. It might also be necessary to run composer install in the CI build; and possibly we need the composer.lock in the shared folder (atm it's not checked in).

Reply:
All done :)